frontend/src/app/application-forms/tree-application-form/tree-application-form.component.ts

```diff
@@ -19,20 +20,22 @@ export class TreeApplicationFormComponent implements OnInit {
   application: any;
   applicationForm: FormGroup;
   maxNumberOfTrees: number;
-  quantityLength: number;
   costPerTree: number;
   apiErrors: any;
+  rulesExpanded = true;
```

Comment:
Since rules will always be displayed is this still necessary?

Reply:
Good catch! removed.